apps/backend/src/routers/users.ts

```diff
@@ -30,33 +37,104 @@ const usersRouter = router({
     /**
      * Loads schedule data for a user that's logged in.
      */
+    getUserAndAccountBySessionToken: procedure
+        .input(z.object({ sessionToken: z.string() }))
+        .query(async ({ input }) => {
+            return await RDS.getAccountUserByToken(db, input.sessionToken);
+        }),
+    getAccountByUid: procedure.input(z.object({ userid: z.string() })).query(async ({ input }) => {
+        return await RDS.getAccountById(db, input.userid);
+    }),
+    getUserByUid: procedure.input(z.object({ userid: z.string() })).query(async ({ input }) => {
+        return await RDS.getUserById(db, input.userid);
+    }),
```

Comment:
Why are these endpoints here? If they're for testing while development, they should be moved to a different, clearly-marked router.